pythermalcomfort/models/bfu_occupational.py

```diff
@@ -0,0 +1,323 @@
+        nude_it = ia_off * corr_ia
+        itr = np.where(
+            (icl > 0) & (icl < 0.093),
+            ((0.093 - icl) * nude_it + icl * clothed_it) / 0.093,
+            clothed_it,
+        )
+        dry = (tsk - t_operative) / np.maximum(itr, 1e-9)
+
+        hedyn = 16.5 * hc
+        corr_ret = np.exp(
+            -0.468 * (vwind - 0.15)
+            + 0.080 * (vwind - 0.15) ** 2
+            - 0.87 * effective_speed
+            + 0.358 * effective_speed**2,
+        )
+        clothed_ret = ret_off * corr_ret
+        nude_dyn = 1.0 / np.maximum(hedyn, 1e-9)
+        ret_r = np.where(
+            icl <= 1e-9,
+            nude_dyn,
+            np.where(
+                (icl > 0) & (icl < 0.093),
+                ((0.093 - icl) * nude_dyn + icl * clothed_ret) / 0.093,
+                clothed_ret,
+            ),
+        )
+
+        emax = np.maximum(psat_tsk - pa, 0.0) / np.maximum(ret_r, 1e-9)
+
+        wettedness = np.divide(
+            swmax,
+            emax,
+            out=np.full_like(emax, np.inf),
+            where=emax > 1e-12,
+        )
+
+        sweff = np.where(
+            wettedness <= 1.0,
+            1.0 - ((wettedness**2) / 2),
+            np.where(
+                wettedness <= 1.7,
+                ((2.0 - wettedness) ** 2) / 2,
+                0.05,
+            ),
+        )
+        sweff = np.clip(sweff, 0.0, 1.0)
+        esweat = swmax * sweff
+        storage = met_arr - (dry + esweat + resp)
+
+        return storage, dry, esweat, sweff, wettedness
+
+    storage_fan, dry_fan, evap_fan, eff_fan, _ = _state(fan_on_arr)
+    storage_off, dry_off, evap_off, eff_off, _ = _state(fan_off_arr)
+
+    delta_storage_raw = storage_fan - storage_off
+    interpretation = np.select(
+        [delta_storage_raw < -50, delta_storage_raw <= 50],
+        [1, 0],
+        default=-1,
+    )
+    if isinstance(interpretation, np.ndarray) and interpretation.shape == ():
+        interpretation = interpretation.item()
+
+    if round_output:
+        storage_fan = np.around(storage_fan, 1)
+        storage_off = np.around(storage_off, 1)
+        delta_storage = np.around(delta_storage_raw, 1)
+        dry_fan = np.around(dry_fan, 1)
+        dry_off = np.around(dry_off, 1)
+        evap_fan = np.around(evap_fan, 1)
+        evap_off = np.around(evap_off, 1)
+        resp = np.around(resp, 1)
+        eff_fan = np.around(eff_fan, 2)
+        eff_off = np.around(eff_off, 2)
+    else:
+        delta_storage = delta_storage_raw
+
+    return BFUOccupationalResult(
+        storage_fan=storage_fan,
+        storage_no_fan=storage_off,
+        delta_storage=delta_storage,
+        dry_heat_fan=dry_fan,
+        dry_heat_no_fan=dry_off,
+        evaporative_heat_fan=evap_fan,
+        evaporative_heat_no_fan=evap_off,
+        respiratory_heat=resp,
+        sweat_efficiency_fan=eff_fan,
+        sweat_efficiency_no_fan=eff_off,
+        interpretation=interpretation,
+    )
+
+
+def BFU_occupational(
```

Comment:
why are there two models with the same name but one is capitalised and one no? we do not need this.